enterprise/templates/enterprise/emails/bulk_enroll_notification.html

```diff
@@ -0,0 +1,371 @@
+<html>
```

Comment:
this template may need some final tuneup but it's quite close to the look we want